src/main/resources/base-target.yaml

```diff
@@ -68,6 +68,11 @@ target:
         - "index.number_of_shards"
         - "index.replication.type"
         - "index.version.upgraded"
+      client:
+        # Number of slices to divide the reindex task
```

Comment:
Can we add a comment here saying: `0 means that OpenSearch will automatically determine the number of slices to use`

Reply:
Updated